playbooks/add_compute.yml

```diff
@@ -0,0 +1,15 @@
+---
+- hosts: airship-openstack-compute-workers
+  gather_facts: yes
+  any_errors_fatal: true
+  roles:
+    - role: airship-configure-worker
+      when: redeploy_osh_only is defined and redeploy_osh_only
```

Comment:
Why this condition is needed ? There is no relation to flag with add compute logic

Reply:
yeah I have fixed that one.